modules/sharedaddy/sharing-sources.php

```diff
@@ -711,7 +719,11 @@ public function has_custom_button_style() {
 	}
 
 	public function display_footer() {
-		$this->js_dialog( $this->shortname, array( 'height' => 350 ) );
+		if ( ! $this->smart ) {
+			$this->js_dialog( $this->shortname, array( 'height' => 350 ) );
+		} else { ?>
```

Comment:
The formatting here is a bit funky and harder to read than it needs to be. I'd suggest either moving `?>` and `<?php` to their own lines, or as prefix/suffix to the `<script>` line.

```php
} else {
	?>
	<script></script>
	<?php
}

// Or...

} else {
	?><script></script><?php
}
```